packages/cactus-plugin-ledger-connector-besu/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM ghcr.io/hyperledger/cactus-cmd-api-server:2022-08-05-7309f2a
-RUN npm install -g yarn@1.22.17
+FROM ghcr.io/hyperledger/cactus-cmd-api-server:2024-03-18-8ddc02d
+RUN npm install -g yarn \
+    && yarn set version 4.1.0 \
+    && yarn config set nodeLinker node-modules
 
 ENV NODE_ENV=production
 ARG NPM_PKG_VERSION=latest
 
-RUN yarn add @hyperledger/cactus-plugin-ledger-connector-besu@${NPM_PKG_VERSION} --production --ignore-engines
+RUN yarn add @hyperledger/cactus-plugin-ledger-connector-besu@${NPM_PKG_VERSION}
```

Comment:
Why are we removing the tags here? @aldousalvarez

Reply:
@jagpreetsinghsasan on yarn version 4 there are no equivalent command to use similar to --production and --ignore-engines, I updated it now to use --exact so it would just add the exact version